action.yml

```diff
@@ -0,0 +1,101 @@
+name: "Sourcery GitHub Action"
+
+branding:
+  icon: 'box'
+  color: 'orange'
+
+description: "A GitHub Action for running Sourcery."
+
+inputs:
+  token:
```

Comment:
🤓 nitpick:

1. Add a link where you can read that token.
Perhaps
https://sourcery.ai/dashboard/profile 
It would be great if we had a page in the docs that explains how the authentication works. But I think it doesn't exist.

2. Add some info that we recommend setting up a separate user for the CI.

Reply:
Again, I think this fits better in our official docs - I created a GitHub issue for it: https://github.com/sourcery-ai/core/issues/2032.